### PR TITLE
execute-this-buffer and execute keys were swapped in README

### PR DESCRIPTION
**Pull request by diogovk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,9 +263,9 @@ The Mix tasks running in a separate `alchemist-mix-mode`, in which the following
 
 | Keybinding | Description |
 |-------------------|-------------|
-|<kbd>C-c a c c</kbd>|Compile the current buffer with the `elixirc` command. `alchemist-compile-this-buffer`|
+|<kbd>C-c a c b</kbd>|Compile the current buffer with the `elixirc` command. `alchemist-compile-this-buffer`|
 |<kbd>C-c a c f</kbd>|Compile the given `FILENAME` with the `elixirc` command. `alchemist-compile-file`|
-|<kbd>C-c a c b</kbd>|Run a custom compile command with `elixirc`. `alchemist-compile`|
+|<kbd>C-c a c e</kbd>|Run a custom compile command with `elixirc`. `alchemist-compile`|
```
Comment (tonini) [inline README.md]: This one needs to be `C - c a c c`

Comment (diogovk) [inline README.md]: You're right, my bad.
It's fixed now.

**Comment by tonini** [comment]
Great! Thank you! :+1:

**Comment by diogovk** [comment]
Actually there was bug in there.
It's actually "C-c a e e" and not "C-c a e c".
I confused with compile.

**Pull request merged**